src/main/webapp/resources/js/apis/projects/create.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import { setBaseUrl } from "../../utilities/url-utilities";
+
```

Comment:
Comments

Reply:
Updated in c82d5ec45d3a927e25f8d033f9fe8b9baadaf580